utils_test.py

```diff
@@ -252,3 +258,52 @@ def test_get_hashable_value(self):
     def test_is_valid_language_code(self):
         self.assertTrue(utils.is_valid_language_code('en'))
         self.assertFalse(utils.is_valid_language_code('unknown'))
+
+    def test_require_valid_name(self):
+        name = 'name'
+        utils.require_valid_name(name, 'name_type')
+
+        name = 0
+        with self.assertRaisesRegexp(Exception, 'name_type must be a string.'):
```

Comment:
The error message seems wrong? name_type is a string right? name is not a string.

Reply:
Good catch! The exception message in utils.py was wrong. It was `'%s must be a string.' % name_type` whereas it should have been `'%s must be a string.' % name` because we are validating name